data/Templates/Hl7v2/Resource/_Observation.liquid

```diff
@@ -125,9 +125,6 @@ Observation_Specimen_ID: A resource Id, used to fill "specimen.reference" proper
                 {% endif -%}
             },
             {
-                {% if OBX.23 and Organization_ID_OBX_23 -%}
-                "reference":"Organization/{{ Organization_ID_OBX_23 }}",
```

Comment:
Are we going to remove this reference?

Reply:
Yes, this reference is duplicate with line #124, maybe it's caused by the error in mapping spreadsheets.

Reply:
Got it. Thanks!